### PR TITLE
Update the runtime build script

### PR DESCRIPTION
**Pull request by joaoalf**
Part of https://github.com/plotly/streambed/issues/13707

### Test plan

- [x] Run `python-generation` and ensure that it generates all supported runtimes.

@gzork Please review.

### CODE REVIEW

File: Dockerfile.heroku-18-libssl1.0 [hunk 1]
```diff
@@ -4,9 +4,10 @@ WORKDIR /app
 ENV WORKSPACE_DIR="/app/builds" \
     S3_BUCKET="lang-python" \
     S3_PREFIX="heroku-18/" \
-    DEBIAN_FRONTEND=noninteractive
+    DEBIAN_FRONTEND=noninteractive \
```
Comment (deepstqte) [inline Dockerfile.heroku-18-libssl1.0:8]: This is good, I wonder if we should also add a note at the top in both files to remind anyways editing either one of them, to make the same change on the other one ?

File: python-generation [hunk 2]
```diff
@@ -21,13 +21,14 @@ main() {
   declare VERSION="$1"
   mkdir -p tmp
 
+  [[ ! -L Dockerfile.heroku-18 ]] && ln -s builds/heroku-18.Dockerfile Dockerfile.heroku-18
```
Comment (deepstqte) [inline python-generation:24]: Huh, how did it work without this ?

**Review by deepstqte — APPROVED** [review]
I left a question and a minor suggestion, other than that, this looks good to me, I'm approving as I'd leave the suggestion up to you @joaoalf :)

File: python-generation (hunk 2 above)
Comment (joaoalf) [reply python-generation:24]: The Python runtime generation is done manually and not very often so, I believe that the last person to do it just copied or linked the file before executing the script.

File: Dockerfile.heroku-18-libssl1.0 (hunk 1 above)
Comment (joaoalf) [reply Dockerfile.heroku-18-libssl1.0:8]: Good call! I will add the note.

Comment (joaoalf) [reply Dockerfile.heroku-18-libssl1.0:8]: Added.

**Comment by joaoalf** [comment]
@gzork Added the comment and one last commit. Still a 💃 ?

**Comment by deepstqte** [comment]
Thanks @joaoalf  yes looks good to me 💃

**Pull request merged**